fluid/DeepASR/data_utils/trans_add_delta.py

```diff
@@ -0,0 +1,91 @@
+#by zhxfl 2018.01.29
+import numpy
+import math
+import copy
+
+
+class TransAddDelta(object):
+    """ add delta of feature data 
+        trans feature for shape(a, b) to shape(a, b * 3)
+    """
+
+    def __init__(self, norder=2, nwindow=2):
+        """ init construction
+            Args:
+                norder: default 2 
```

Comment:
Please explain the meaning of arguments.

Reply:
done